app/controllers/cards_controller.rb

```diff
@@ -37,6 +37,15 @@ def destroy
     redirect_to board_cards_path
   end
 
+  def learning
+    @cards = @board.cards
+    if @cards.empty?
+      redirect_to board_cards_path(@board)
```

Comment:
maybe we should add some notice ? smth like 'There is nothing to learn' or 'Add cards before starting learning'